packages/cli/src/index.ts

```diff
@@ -1 +1 @@
-export * from './lib';
+export * from './lib/index.js';
```

Comment:
I find it a little strange that we have a barrel that litterly reexport another barrel